src/ddmd/mars.d

```diff
@@ -1164,7 +1163,7 @@ Language changes listed by -transition=id:
     Module._init();
     Target._init();
     Expression._init();
-    objc_tryMain_init();
+    objc.initialize();
```

Comment:
The `initialize` function has been moved into the `Objc` interface as a static method and renamed to `_init`. At the call site it's now: `Objc._init();`.